index.js

```diff
@@ -723,5 +723,36 @@ module.exports = input => {
 		};
 	}
 
+	// File Type Box (https://en.wikipedia.org/wiki/ISO_base_media_file_format)
+	if (check([0x66, 0x74, 0x79, 0x70], {offset: 4})) {
+		if (check([0x6D, 0x69, 0x66, 0x31], {offset: 8})) {
+			return {
+				ext: 'heic',
+				mime: 'image/heif'
```

Comment:
Shouldn't this be `image/heic`?

https://infogalactic.com/info/High_Efficiency_Image_File_Format